input/includes/PATCH_EpisodeOfCare.md

```diff
@@ -42,76 +38,43 @@ __Response__
     {
       "url": "http://ehealth.sundhed.dk/fhir/StructureDefinition/ehealth-episodeofcare-caremanagerOrganization",
       "valueReference": {
-        "reference": "https://organization.cit-careplan-2740.local/fhir/Organization/83136"
+        "reference": "https://organization.cit-careplan-2818.local/fhir/Organization/86577"
       }
-    },
-    {
-      "url": "http://ehealth.sundhed.dk/fhir/StructureDefinition/ehealth-teamHistory",
-      "extension": [
-        {
-          "url": "careTeam",
-          "valueReference": {
-            "reference": "https://organization.cit-careplan-2740.local/fhir/CareTeam/51193"
-          }
-        },
-        {
-          "url": "period",
-          "valuePeriod": {
-            "start": "2024-04-04T08:59:26+00:00",
-            "end": "2024-04-05T08:59:27+00:00"
-          }
-        }
-      ]
-    },
-    {
-      "url": "http://ehealth.sundhed.dk/fhir/StructureDefinition/ehealth-teamHistory",
-      "extension": [
-        {
-          "url": "careTeam",
-          "valueReference": {
-            "reference": "https://organization.cit-careplan-2740.local/fhir/CareTeam/81791"
-          }
-        },
-        {
-          "url": "period",
-          "valuePeriod": {
-            "start": "2024-04-05T08:59:27+00:00"
-          }
-        }
-      ]
     }
   ],
   "status": "active",
   "statusHistory": [
+    {
+      "status": "finished",
+      "period": {
+        "start": "2024-07-29T13:13:34+00:00",
+        "end": "2024-07-29T13:13:34+00:00"
+      }
+    },
     {
       "status": "active",
       "period": {
-        "start": "2024-04-05T08:59:26+00:00"
+        "start": "2024-07-29T13:13:34+00:00"
       }
     }
   ],
   "diagnosis": [
     {
       "condition": {
-        "reference": "https://careplan.cit-careplan-2740.local/fhir/Condition/1672"
+        "reference": "https://careplan.cit-careplan-2818.local/fhir/Condition/1546"
       },
       "rank": 1
     }
   ],
   "patient": {
-    "reference": "https://patient.cit-careplan-2740.local/fhir/Patient/25415"
+    "reference": "https://patient.cit-careplan-2818.local/fhir/Patient/97295"
   },
   "managingOrganization": {
-    "reference": "https://organization.cit-careplan-2740.local/fhir/Organization/83136"
+    "reference": "https://organization.cit-careplan-2818.local/fhir/Organization/86577"
   },
   "period": {
     "start": "1970-01-01T00:00:01+00:00",
     "end": "1970-01-01T00:01:40+00:00"
-  },
-  "team": [
```

Comment:
Do we know why this has been removed?

Reply:
Can be ignored, as I understand it the examples are based on tests and I guess it might not be the same test every time?

Reply:
Same as above